### PR TITLE
Replace nonce with challenge, add jws and proofPurpose.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
Update LD Proofs examples to match @dlongley's PR https://github.com/w3c/vc-data-model/pull/619

### CODE REVIEW

File: test/vc-data-model-1.0/input/example-5-bad-proof-missing-type.jsonld [hunk 1]
```diff
@@ -16,8 +16,9 @@
   },
   "proof": {
     "created": "2017-06-18T21:19:10Z",
-    "creator": "https://example.com/jdoe/keys/1",
-    "nonce": "c0ae1c8e-c7e7-469f-b252-86e6a0e7387e",
-    "signatureValue": "BavEll0/I1zpYw8XNi1bgVg/sCneO4Jugez8RwDg/+MCRVpjOboDoe4SxxKjkCOvKiCHGDvc4krqi6Z1n0UfqzxGfmatCuFibcC1wpsPRdW+gGsutPTLzvueMWmFhwYmfIFpbBu95t501+rSLHIEuujM/+PXr9Cky6Ed+W3JT24="
+    "proofPurpose": "authentication",
```
Comment (dlongley) [inline test/vc-data-model-1.0/input/example-5-bad-proof-missing-type.jsonld]:
```suggestion
    "proofPurpose": "assertionMethod",
```

File: test/vc-data-model-1.0/input/example-5-bad-proof-missing-type.jsonld [hunk 2]
```diff
@@ -16,8 +16,9 @@
...
+    "verificationMethod": "https://example.com/jdoe/keys/1",
+    "challenge": "c0ae1c8e-c7e7-469f-b252-86e6a0e7387e",
```
Comment (dlongley) [inline test/vc-data-model-1.0/input/example-5-bad-proof-missing-type.jsonld]:
```suggestion
```

File: test/vc-data-model-1.0/input/example-5-bad-proof-missing-type.jsonld (hunk 1 above)
Comment (dlongley) [reply test/vc-data-model-1.0/input/example-5-bad-proof-missing-type.jsonld]: This is on a VC, so the purpose of the proof is as a method of assertion, it is not for authentication.

File: test/vc-data-model-1.0/input/example-5-bad-proof-missing-type.jsonld (hunk 2 above)
Comment (dlongley) [reply test/vc-data-model-1.0/input/example-5-bad-proof-missing-type.jsonld]: `challenge` is for authentication proofs, this is an assertion method proof.

**Review by dlongley — CHANGES_REQUESTED** [review]

File: test/vc-data-model-1.0/input/example-5.jsonld [hunk 3]
```diff
@@ -17,7 +17,9 @@
   "proof": {
     "type": "RsaSignature2018",
     "created": "2017-06-18T21:19:10Z",
+    "proofPurpose": "authentication",
     "verificationMethod": "https://example.com/jdoe/keys/1",
-    "jws": "BavEll0/I1zpYw8XNi1bgVg/sCneO4Jugez8RwDg/+MCRVpjOboDoe4SxxKjkCOvKiCHGDvc4krqi6Z1n0UfqzxGfmatCuFibcC1wpsPRdW+gGsutPTLzvueMWmFhwYmfIFpbBu95t501+rSLHIEuujM/+PXr9Cky6Ed+W3JT24="
+    "challenge": "c0ae1c8e-c7e7-469f-b252-86e6a0e7387e",
+    "jws": "eyJhbGciOiJSUzI1NiIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..TCYt5XsITJX1CxPCT8yAV-TVkIEq_PbChOMqsLfRoPsnsgw5WEuts01mq-pQy7UJiN5mgRxD-WUcX16dUEMGlv50aqzpqh4Qktb3rk-BuQy72IFLOqV0G_zS245-kronKb78cPN25DGlcTwLtjPAYuNzVBAh4vGHSrQyHUdBBPM"
```
Comment (dlongley) [inline test/vc-data-model-1.0/input/example-5.jsonld:22]: Same issues as above.

File: test/vc-data-model-1.0/input/example-5-bad-proof-missing-type.jsonld (hunk 2 above)
Comment (dmitrizagidulin) [reply test/vc-data-model-1.0/input/example-5-bad-proof-missing-type.jsonld]: Whoops, fixed. Thanks!

File: test/vc-data-model-1.0/input/example-5.jsonld (hunk 3 above)
Comment (dmitrizagidulin) [reply test/vc-data-model-1.0/input/example-5.jsonld:22]: Thanks, fixed.

**Review by dlongley — APPROVED** [review]

**Pull request merged**